src/dbnode/storage/series/buffer.go

```diff
@@ -54,6 +54,8 @@ const (
 var (
 	timeZero           time.Time
 	errIncompleteMerge = errors.New("bucket merge did not result in only one encoder")
+	errTooManyEncoders = xerrors.NewInvalidParamsError(errors.New("too many encoders per block"))
+	logger, _          = zap.NewProduction()
```

Comment:
why create this here?

Reply:
Good catch... not sure where that came from.